diff --git a/setup.py b/setup.py
index d5108f1..35b7bf3 100644
--- a/setup.py
+++ b/setup.py
@@ -120,11 +120,12 @@ else:
         )
     ) + [os.path.join(SKIA_OUT_PATH, 'libskia.a')]
     EXTRA_COMPILE_ARGS = [
-        '-std=c++14',
+        '-std=c++17',
         '-fvisibility=hidden',
         '-Wno-attributes',
         '-fdata-sections',
         '-ffunction-sections',
+        '-DGR_TEST_UTILS',
     ]
     EXTRA_LINK_ARGS = [
         '-Wl,--gc-sections',
