commit ed0847f5884b219409c731aad104927b64e08cc8
Author: Lein Matsumaru <appleprincess@appleprincess.io>
Date:   Sun Jan 8 14:12:51 2023 +0000

    b

diff --git a/bench/nanobench.h b/bench/nanobench.h
index d1f94bc..ffd0125 100644
--- a/bench/nanobench.h
+++ b/bench/nanobench.h
@@ -9,6 +9,7 @@
 #define nanobench_DEFINED
 
 #include "bench/Benchmark.h"
+#include "include/core/SkBitmap.h"
 #include "include/core/SkImageInfo.h"
 #include "include/core/SkSurface.h"
 #include "include/core/SkTypes.h"
