commit 7783937eb847ff3d79d2be5e5e20b3d75862942a
Author: Lein Matsumaru <appleprincess@appleprincess.io>
Date:   Sun Jan 8 11:37:30 2023 +0000

    b

diff --git a/third_party/freetype2/BUILD.gn b/third_party/freetype2/BUILD.gn
index 8daf603..3cd083c 100644
--- a/third_party/freetype2/BUILD.gn
+++ b/third_party/freetype2/BUILD.gn
@@ -10,7 +10,7 @@ declare_args() {
   skia_use_system_freetype2 =
       (is_official_build || !(is_android || sanitize == "MSAN")) &&
       !is_fuchsia && !is_wasm
-  skia_system_freetype2_include_path = "/usr/include/freetype2"
+  skia_system_freetype2_include_path = "@INCLUDE_DIR_FREETYPE2@"
   skia_system_freetype2_lib = "freetype"
 
   skia_use_freetype_woff2 = is_wasm
diff --git a/third_party/harfbuzz/BUILD.gn b/third_party/harfbuzz/BUILD.gn
index 6838f73..8271930 100644
--- a/third_party/harfbuzz/BUILD.gn
+++ b/third_party/harfbuzz/BUILD.gn
@@ -12,7 +12,7 @@ declare_args() {
 
 if (skia_use_system_harfbuzz) {
   system("harfbuzz") {
-    include_dirs = [ "/usr/include/harfbuzz" ]
+    include_dirs = [ "@INCLUDE_DIR_HARFBUZZ@" ]
     libs = [ "harfbuzz" ]
     if (skia_pdf_subset_harfbuzz) {
       libs += [ "harfbuzz-subset" ]
